### DISCUSSION TITLE
Welcome to .github Discussions!

### DISCUSSION THREAD

**Discussion by rickypazd** [DHM-DISTRIBUIDORA/.github#22] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rickypazd** [comment]
Hola estamos probando aca esta funcionalidad para tratar de resolver el problema de comunicacion.

**Comment by ruddypazd** [comment]
A mi me salio en mi inicio
